src/test_util/components.rs

```diff
@@ -72,6 +72,16 @@ lazy_static! {
             "component_sent_event_bytes_total",
         ],
     };
+    /// The component test specification for components with multiple outputs
+    pub static ref COMPONENT_MULTIPLE_OUTPUTS_TESTS: ComponentTests = ComponentTests {
+        events: &["EventsSent"],
+        tagged_counters: &[
+            "component_sent_events_total",
+            "component_sent_event_bytes_total",
+        ],
+        untagged_counters: &[
+        ],
+    };
 }
 
 impl ComponentTests {
```

Comment:
Should this be modified to ensure the counters have the `output` tag?

Reply:
What modifications did you have in mind here? We currently pass in `output` when calling `COMPONENT_MULTIPLE_OUTPUTS_TESTS.assert()`.

Reply:
Okay, that should be good enough.